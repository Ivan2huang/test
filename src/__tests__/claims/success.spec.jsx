/* eslint-disable react/prop-types */

import React from 'react';
import { render } from '@testing-library/react';

import SuccessPage from '../../pages/claims/success';

jest.mock('../../layouts/withLayoutProvider', () => Component => () => (
  <div>
    <header>Layout Header</header>
    <Component />
  </div>
));

jest.mock('../../modules/claims/make-claim', () => ({
  MakeClaimSuccess: ({ children }) => (
    <div>
      <header>Success Component</header>
      {children}
    </div>
  ),
}));

describe('Make claim Page', () => {
  it('should match snapshot', () => {
    const { container } = render(<SuccessPage />);

    expect(container).toMatchSnapshot();
  });
});
