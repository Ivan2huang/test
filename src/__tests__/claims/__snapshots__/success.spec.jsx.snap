// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make claim Page should match snapshot 1`] = `
<div>
  <div>
    <header>
      Layout Header
    </header>
    <div>
      <header>
        Success Component
      </header>
    </div>
  </div>
</div>
`;
